katacomb/katacomb/aips_export.py

```diff
@@ -271,34 +271,37 @@ def export_calibration_solutions(uv_files, kat_adapter, mfimage_params, telstate
     for si, uv_file in enumerate(uv_files):
         try:
             with uv_factory(aips_path=uv_file, mode='r') as uvf:
-                try:
-                    log.info('Extracting phase only self-calibration from AIPS SN: %d' % (pSN))
-                    uvf.attach_table("AIPS SN", pSN)
+                log.info('Extracting self-calibration solutions from %s' % uv_file)
```

Comment:
Style: your loggers can be simplified to this (variables as function arguments instead of behind a %):
```Python
log.info('Extracting self-calibration solutions from %s', uv_file)
```

Reply:
Ah. OK. I might go through the lot and fix it any place else its happening as well - and I'll put it in a different commit so you don't have to look at it.